onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -213,17 +213,25 @@ def aten_celu(self: FLOAT, alpha: float = 1.0) -> FLOAT:
     return op.Celu(self, alpha=alpha)  # op.Celu only support float32
 
 
+@torch_op("aten::col2im")
 def aten_col2im(
-    self: TensorType,
+    self: TReal,
     output_size: INT64,
     kernel_size: Sequence[int],
-    dilation: Sequence[int],
-    padding: Sequence[int],
-    stride: Sequence[int],
-) -> TensorType:
+    dilation: Sequence[int] = [1, 1],
+    padding: Sequence[int] = [0, 0, 0, 0],
+    stride: Sequence[int] = [1, 1],
```

Comment:
I don't find these three having default value on PyTorch. Am I missing it?

Reply:
I agree. We don't need to fill in defaults when the aten op does not specify it